server/src/main/java/org/elasticsearch/common/blobstore/EndpointStats.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+package org.elasticsearch.common.blobstore;
+
+import org.elasticsearch.common.io.stream.StreamInput;
+import org.elasticsearch.common.io.stream.StreamOutput;
+import org.elasticsearch.common.io.stream.Writeable;
+import org.elasticsearch.xcontent.ToXContentObject;
+import org.elasticsearch.xcontent.XContentBuilder;
+
+import java.io.IOException;
+import java.util.Objects;
+
+/**
+ * The number of calls to an endpoint
+ *
+ * @param operations The number of calls
+ * @param requests The number of calls (including retries)
+ * @param legacyValue The number used before the migration to separate counts (sometimes operations, sometimes requests, for temporary BWC)
```

Comment:
If perf are cool with cutting over to request counts for Azure abruptly we can NOT do the `legacyValue` stuff. For BWC we can just take the single count we get and use it for `requests` and `operations`. As it'll only be momentary during an upgrade anyhow.

Reply:
Does perf only use the internal blob store stats API? Are they interested in `requests` or `operations` or both? If this legacyValue is only for making the internal API BWC, I think it is not really worth it unless you are using it for the [external API](https://www.elastic.co/guide/en/elasticsearch/reference/current/get-repositories-metering-api.html) as well.

Reply:
Perf have agreed it's OK to just cut over to the new values. I've removed `legacyValue` now.